app/scripts/lib/stores/extension-store.ts

```diff
@@ -1,20 +1,15 @@
 import browser from 'webextension-polyfill';
 import log from 'loglevel';
-import {
-  type MetaMaskStateType,
-  type MetaMaskStorageStructure,
-  BaseStore,
-} from './base-store';
+import type { MetaMaskStorageStructure, BaseStore } from './base-store';
 
 /**
  * An implementation of the MetaMask Extension BaseStore system that uses the
  * browser.storage.local API to persist and retrieve state.
  */
-export default class ExtensionStore extends BaseStore {
+export default class ExtensionStore implements BaseStore {
   isSupported: boolean;
 
   constructor() {
-    super();
```

Comment:
removed super because we no longer extend a `BaseStore`  class, but instead of implement a `BaseStore` type